app/models/internal/impl/DefaultAlert.java

```diff
@@ -43,51 +40,21 @@ public UUID getId() {
         return _id;
```

Comment:
Don't we have owner/organization in the model? Shouldn't alert be associated with one?

Reply:
I wrestled with that quite a bit.  Now, it's up to the repositories to tie them together.  In the data models, they're likely together.  In the query layer, the repositories are handed an org id and an alert id separately.